doc/api/crypto.markdown

```diff
@@ -651,13 +651,9 @@ Generates cryptographically strong pseudo-random data. Usage:
     });
 
     // sync
-    try {
-      var buf = crypto.randomBytes(256);
-      console.log('Have %d bytes of random data: %s', buf.length, buf);
-    } catch (ex) {
-      // handle error
-      // most likely, entropy sources are drained
-    }
+    // Implementation of handling errors is recommended.
```

Comment:
Does this mean there should be a try/catch? `randomBytes()` will never throw anymore, right?